docs/guide/personalization/recommendation_client.md

```diff
@@ -73,7 +73,7 @@ Allow public HTTP(S) access to the recommendation bundle API (`<yourdomain>/api/
     If your organization's policy is to use BASIC AUTH on the API interfaces,
     you need to add specific configuration.
 
-You can define access restrictions on your site in `app/config/security.yml`.
+You can define access restrictions on your site in `app/config/security.yaml`.
```

Comment:
If this refers to the file in meta, the path must also be changed, see https://github.com/ezsystems/ezplatform-ee/tree/master/config
```suggestion
You can define access restrictions on your site in `config/packages/security.yaml`.
```